Dockerfile

```diff
@@ -22,7 +22,9 @@ ENV OSX_CROSS_PATH=/osxcross
 FROM base AS osx-sdk
 ARG OSX_SDK
 ARG OSX_SDK_SUM
-ADD https://s3.dockerproject.org/darwin/v2/${OSX_SDK}.tar.xz "${OSX_CROSS_PATH}/tarballs/${OSX_SDK}.tar.xz"
+# ADD https://s3.dockerproject.org/darwin/v2/${OSX_SDK}.tar.xz "${OSX_CROSS_PATH}/tarballs/${OSX_SDK}.tar.xz"
+# FIXME: upload to s3.dockerproject.org
+ADD https://github.com/ndeloof/golang-cross/raw/113fix/${OSX_SDK}.tar.xz "${OSX_CROSS_PATH}/tarballs/${OSX_SDK}.tar.xz"
```

Comment:
I used the SDK https://github.com/phracker/MacOSX-SDKs, but unfortunately it lacks the necessary headers file. 
I got some error like this:

```
testing o32-clang ... osxcross: error: cannot find libc++ headers
osxcross: error: while detecting target
```
The full job can be found here https://gitlab.com/taobeier/golang-cross/-/jobs/308408417

Later I chose the SDK build by @ndeloof  It works well. https://gitlab.com/taobeier/golang-cross/-/jobs/309123113

I think we shoud upload new version SDK to s3.dockerproject.org @thaJeztah